charts/globus-compute/README.md

```diff
@@ -0,0 +1,122 @@
+# Globus Compute Endpoint 
+
+This SLATE application will deploy a functioning Kubernetes endpoint into your SLATE cluster and will launch worker pods with a specified container image into a namespace.
+
+## How to Use
+
+There are two required values to specify in the `values.yaml` file, `endpointUUID` and authentication. 
```

Comment:
I missed this line. Instead of:

```in the`values.yaml` file```

can we say:

```in the application configuration```

Reply:
Done.